server/database/migrations/000004_character.sql

```diff
@@ -1,21 +1,22 @@
 DROP TYPE IF EXISTS "character.STATE_TYPE";
 CREATE TYPE character.STATE_TYPE AS ENUM(
     'idle',
-    'walking'
+    'walking',
+    'collecting_resource'
 );
 
 CREATE TABLE character.instance(
-       name VARCHAR(18) NOT NULL,
+       name TEXT NOT NULL,
```

Comment:
Same here, either a domain type or a check constraint, since this will be user input.